File: app/src/main/java/com/artmurka/artmurkaapp/Model/Pojo/ItemList/WishList/EntryWeight.java

package com.artmurka.artmurkaapp.Model.Pojo.ItemList.WishList;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class EntryWeight {

    @SerializedName("weight_raw")
    @Expose
    private String weightRaw;
    @SerializedName("weight")
    @Expose
    private Long weight;

    public String getWeightRaw() {
        return weightRaw;
    }

    public void setWeightRaw(String weightRaw) {
        this.weightRaw = weightRaw;
    }

    public Long getWeight() {
        return weight;
    }

    public void setWeight(Long weight) {
        this.weight = weight;
    }

}

File: app/src/main/java/com/artmurka/artmurkaapp/data/model/pojo/itemlist/wishList/EntryWeight.kt
package com.artmurka.artmurkaapp.data.model.pojo.itemlist.wishList

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class EntryWeight {

    @SerializedName("weight_raw")
    @Expose
    var weightRaw: String? = null
    @SerializedName("weight")
    @Expose
    var weight: Long? = null

}
